src/Mod/PartDesign/App/FeatureHole.cpp

```diff
@@ -447,6 +448,36 @@ const double Hole::metricHoleDiameters[36][4] =
         { 68.0,     70.0,  	77.0,   78.0}
 };
 
+const Hole::UTSClearanceDefinition Hole::UTSHoleDiameters[22] =
+{
+    /* UTS clearance hole diameters according to ASME B18.2.8 */

```

Comment:
I suggest a clarifying note here: I believe that standard actually lists a min and max hole size for each case, in inches. So if you used something like the Amesweb calculator to get the single metric values you might state that here, since it's not _quite_ exactly per the standard itself. And for the record I did double-check them, and they do match the Amesweb results.

Reply:
Thanks for the comment. Yes, the standards defines actually a drill bit number (that is in turn standardized in another ASME norm). So the minimal clearance is the diameter of that drill bit. So we use this minimum as the theoretical exact diameter. This is also done in the ISO norm. That in reality, the resulting diameter of a drilling is wider, is clear and thus the ASME norm also defines the maximum allowed diameter, de-facto as tolerance.

I clarified this now in the comment.